### PR TITLE
options and arguments have been changed

### PR DESCRIPTION
**Pull request by tonicbupt**

### CODE REVIEW

**Comment by yujunz** [comment]
Do we have to update it manually every time a new version is released? Better to put something invariable or automate the updates.

**Comment by tonicbupt** [comment]
Maybe just `juicesync -h` will do...

File: README.md [hunk 1]
```diff
@@ -52,18 +52,33 @@ go get -u github.com/juicedata/juicesync
...
-    	the start of keys to sync
-  -v	turn on debug log
-  --help show the usage
+$ juicesync -h
+NAME:
+   juicesync - Usage: juicesync [options] SRC DST
+    SRC and DST should be [NAME://][ACCESS_KEY:SECRET_KEY@]BUCKET.ENDPOINT[/PREFIX]
+
+USAGE:
+   juicesync [global options] command [command options] [arguments...]
```
Comment (davies) [inline README.md]: we don't support sub-command here

Comment (yujunz) [reply README.md]: We do support the sub-command `help`

Comment (tonicbupt) [reply README.md]: it's actually generated by the module... but it's a good idea to make it's update automatically

**Comment by yujunz** [comment]
Can we get this merged to release a new version? Maybe bump to 0.1.0 or 1.0.0

**Pull request merged**